helm-charts/example-konk-service/templates/konk-service.yaml

```diff
@@ -6,6 +6,6 @@ spec:
   konk:
     name: example-konk
   service:
-    name: test
+    name: {{ .Release.Name }}-example-apiserver
```

Comment:
I think you can change this to `runner-example-apiserver`.

Reply:
`runner-example-apiserver` is only valid in Github Actions, where the username is `runner`. I was hoping for a name that would work everywhere, but this isn't quite it. Locally, this turns into `thayward-example-konk-service-example-apiserver`, which isn't right either.

Reply:
This value should be overridden when the KonkService CR is used in any service chart, so I think for this example CR, we can just set a static `service.name` as `runner-example-apiserver`, since the Github action is currently the only one who uses this example CR.

I'm making some changes in the README to include info about the new charts which will elaborate more on the actual values required.

Reply:
I changed it, but also changed the makefile to always call it `runner-example-apiserver` so that it works everywhere.